lime/explanation.py

```diff
@@ -324,4 +331,9 @@ def jsonize(x):
         ''' % (random_id, predict_proba_js, predict_value_js, exp_js, raw_js)
         out += u'</body></html>'
 
+        if (dark_mode):
+            out = out.replace("\"black\"", "\"white\"")
+            out = out.replace("all: initial;", "all: initial; color: white;")
+            out = re.sub(r"svg.append\('text(((?!fill).)*);", r"svg.append('text\1.style('fill', 'white');", out)
```

Comment:
This regex finds all svg texts (which don't adhere to any css or html color styling) for which a color is _not_ specified (to avoid touching the purposefully colorful words) and decorates them with another style (fill: white)